docs/attributes-registry/build.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# Build
+
+## Build Attributes
+
+This group describes attributes specific to builds (ci).
+
+| Attribute                | Type   | Description                                  | Examples                                                           | Stability                                                        |
+| ------------------------ | ------ | -------------------------------------------- | ------------------------------------------------------------------ | ---------------------------------------------------------------- |
+| `build.artifact.id`      | string | The id of the build artifact.                | `120912`                                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `build.artifact.name`    | string | The name of the build artifact.              | `golang-binary-amd64-v0.1.0`; `docker-image-amd64-v0.1.0`          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Maybe a type attribute would be useful (e.g. image, binary, bundle) not to rely on naming conventions as in the examples.

Reply:
I like it, something like these package types :

* [JFrog Supported package types](https://jfrog.com/help/r/jfrog-artifactory-documentation/supported-package-types)
* [Nexus Supported formats](https://help.sonatype.com/en/formats.html)